docs/experience/configure-requirements.md

```diff
@@ -0,0 +1,34 @@
+title: Configure Requirements
+description: Guide on how to configure Sematext Experience
+
+Different websites will have different performance requirements. We recommend a time threshold of 2s for page loads and HTTP requests and this is also the default we provide. You can change these settings in the Configure Requirements page. Time thresholds are defined in milliseconds.
+
+### Page Loads and HTTP requests
+
+To configure time thresholds for page loads and requests, click on Configure Requirements in the app sidebar menu and adjust the "Page Load Time Threshold". To configure time threshold for requests, adjust the "Request Time Threshold". Below you can see a screenshot of both Page Load time threshold and Request Time Threshold set to two seconds.
+
+<img
+  class="content-modal-image"
+  alt="Create Experience Configure Form"
+  src="../../images/experience/configure-1.png"
+  title="Create Experience Configure Form"
+  width=645
+  height=393
+/>
+
+### URL Groups
+
+To configure time thresholds for specific URL group, click on Configure Requirements in the app sidebar menu and adjust the time threshold input next to each url group.
```

Comment:
To configure time thresholds for specific URL group, click on Configure Requirements in the app sidebar menu and adjust the time threshold input next to each url group.
->
To configure time thresholds for specific URL groups, click on Configure Requirements in the App sidebar menu and adjust the time threshold input next to each URL group.